test/Microsoft.AspNetCore.Razor.Language.Test/DefaultChildTagDescriptorBuilderTest.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.Language
+{
+    public class DefaultChildTagDescriptorBuilderTest
+    {
+        [Fact]
+        public void Build_DisplayNameIsName()
+        {
+            // Arrange
+            var builder = new DefaultChildTagDescriptorBuilder(null);
+            builder.Name = "foo";
+
+            // Act
+            var descriptor = builder.Build();
+
+            // Assert
+            Assert.Equal("foo", descriptor.DisplayName);
+        }
+    }
```

Comment:
Didn't add additional builder tests because the rest of the features of the child tag descriptor are overly tested by all of our existing integration / unit tests.